src/component/label.js

```diff
@@ -15,13 +15,15 @@ const balanceStyles = StyleSheet.create({
   numeral: {
     fontFamily: 'WorkSans ExtraLight',
     fontSize: font.sizeXXXL,
-    lineHeight: font.lineHeightXXXL,
+    lineHeight: null,
+    marginLeft: 10,
+    marginRight: 10,
   },
   unit: {
     fontFamily: 'WorkSans Regular',
     fontSize: font.sizeL,
-    lineHeight: 60,
-    marginLeft: 10,
+    paddingTop: 5,
+    marginRight: 10,
```

Comment:
This break the alignment of the two labels at the top:

![image](https://user-images.githubusercontent.com/1374174/51888106-8ac35800-23c8-11e9-868f-8713f3fa8f31.png)

![image](https://user-images.githubusercontent.com/1374174/51888120-9f075500-23c8-11e9-90e3-6277da679d53.png)

Reply:
Ahh yeah it's so tricky. When I make them start out at the same height, the unit shifts down awkwardly when the amount gets smaller in response to screen width. Working on this..

Reply:
@tanx Check it out now. A remaining issue is that when you have a balance with a lot of digits, the unit stops aligning at the top because its `marginTop` is hard coded to 15. 

The alternative I can think of would be a potentially complex calculation using DOM manipulation/`measure`: https://facebook.github.io/react-native/docs/direct-manipulation#measurecallback to dynamically set the `marginTop` instead. Maybe that's better? It seems like a bit of an edge case so I thought this might be acceptable.

Reply:
> The alternative I can think of would be a potentially complex calculation using DOM manipulation/`measure`: https://facebook.github.io/react-native/docs/direct-manipulation#measurecallback to dynamically set the `marginTop` instead.

Could be worth a try.

Also worth noting the current solution works for fiat since the `$` symbol is already part of the string:
![image](https://user-images.githubusercontent.com/1374174/52198120-6538c180-2894-11e9-8e02-6afae0e6628e.png)

Just doesn't work for BTC:
![image](https://user-images.githubusercontent.com/1374174/52198177-8bf6f800-2894-11e9-9f06-abd4be16614f.png)

Reply:
@tanx Are you sure the above UI bug isn't because you removed the `width` i set here, that you suggested I remove? https://github.com/lightninglabs/lightning-app/pull/890#discussion-diff-251703865R109

Because for me it scales when the width is set:
![51284433_572653783208775_5728527671739023360_n](https://user-images.githubusercontent.com/10647624/52374328-d0f46900-2a11-11e9-8ecb-4a82e65eb908.jpg)

Reply:
It still clearly has problems but at least the font size of the number scales :sweat_smile:

Reply:
> Could be worth a try.

Okay, I'll try that solution today.

Reply:
@tanx Actually, this would be easy to solve without messing with DOM nodes if we had a monospace font (because we could just calculate `marginTop` for the unit based on the screen width and the number of characters in the balance).

Permission to ask Lee again?